vocabularies/gsq-roles.ttl

```diff
@@ -150,6 +152,34 @@ isorole:publisher a skos:Concept ;
     skos:prefLabel "Publisher"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/gsq-roles>.
         
+gsqr:data-creator
```

Comment:
Data Creator is probably conceptually the same as isorole:originator. In DES, we have a defined IM term of Data Creator. 

Thoughts on adding a skos:closeMatch or skos:exactMatch from data-creator to originator? @UbuntuScott

Reply:
You can add Data Creator as an altLabel to isorole:originator.

Reply:
I've updated data-creator to have a closeMatch to Originator, which is in addition to the closeMatch for Author.

I spoke with @UbuntuScott and we feel the definition for Data-Creator is sufficiently different to the Author and Originator - since they are _responsible for creating and updating data, products and metadata and **storing in an appropriate system**_.

Are you OK with that logic?